### PR TITLE
installing  Prometheus operator on ibm cloud

### PR DESCRIPTION
**Pull request by Sumithkr**
[Prometheus Operator.docx](https://github.com/prometheus-operator/prometheus-operator/files/5825864/Prometheus.Operator.docx)
Add this document on documentation form it has all instruction how to install prometheus on ibm cloud

### CODE REVIEW

**Comment by coderanger** [comment]
Thank you for your contribution. It was a bit hard to follow, is there any special procedure needed on IBM Cloud? Our standard recommendation is to use the kube-prometheus manifests, which include prometheus-operator as well as many related tools, and that should work as-is on any Kubernetes cluster (give or take a few of the alerts that requires scraping control plane services that some hosted Kubernetes don't expose).

**Review by Sumithkr — COMMENTED** [review]
[prometheus-operator.docx](https://github.com/prometheus-operator/prometheus-operator/files/5825879/prometheus-operator.docx)
adding this document on documentation form

**Review by paulfantom — CHANGES_REQUESTED** [review]
From the look at the document the described procedure doesn't install just prometheus-operator but it is some sort of wrapper on kube-prometheus.

**Comment by simonpasquier** [comment]
This is something that would be better contributed to the cloud provider's documentation directly. If we start having provider-specific docs in this repository, I can see how they would rot pretty quickly. None of the maintainers would be able to ensure the instructions are still valid but we would still have to deal with end-users hitting issues.

**Comment by s-urbaniak** [comment]
Agreed on the rationale expressed by @simonpasquier, hence closing out.

**Comment by Sumithkr** [comment]
We update our document  please check it
[Prometheus.Operator.docx](https://github.com/prometheus-operator/prometheus-operator/files/5857431/Prometheus.Operator.docx)
